### PR TITLE
documentation for user invoked ownership transfer

### PR DESCRIPTION
**Pull request by wiswedel**
Signed-off-by: Sascha Wiswedel <sascha.wiswedel@nextcloud.com>

### CODE REVIEW

**Comment by wiswedel** [comment]
no further backports, function came with 18

File: user_manual/files/transfer_ownership.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+========
+Transfer Ownership
+========
```
Comment (nickvergessen) [inline user_manual/files/transfer_ownership.rst:3]:
```suggestion
==================
Transfer Ownership
==================
```

**Pull request merged**